### ISSUE TITLE
[Docs] nn.init.._calculate_fan_in_and_fan_out()

### ISSUE THREAD

**Issue by tejan-rgb** [pytorch/pytorch#57712]
## 📚 Documentation

nn.init.._calculate_fan_in_and_fan_out() is missing from the documentation in torch.nn.init

**Comment by jbschlosser** [comment]
This is true, but we generally don't document internal functions (i.e. those beginning with an underscore).

**Comment by tejan-rgb** [comment]
But, it is a useful thing. And it was in the docs before.

**Comment by VitalyFedyunin** [comment]
I would agree with @jbschlosser that we do not document or encourage usage of underscore functions.

**Comment by jbschlosser** [comment]
Perhaps there is a case to be made for adding this function to utils in the public API?

**Comment by tejan-rgb** [comment]
yes, please.
